python/ray/remote_function.py

```diff
@@ -254,6 +252,15 @@ def _remote(self, args=None, kwargs=None, **task_options):
         worker = ray._private.worker.global_worker
         worker.check_connected()
 
+        # We cannot do this when the function is first defined, because we need
```

Comment:
Does this mean the before this change, it was already broken? I think when the remote function is first created, ray should automatically call ray.init

Reply:
ray calls ray.init when the function is first invoked, not defined